networking/ipv6.md

```diff
@@ -1,145 +1,134 @@
-  possible to configure a dual-stack environment.
-- The steps and setup here have not been tested against an existing IPv4
-  cluster and are intended only for new clusters.
+- **IPv6 only**
+   
+  Workloads can communicate over IPv6, initiate connections to IPv6 services, and terminate incoming IPv6 connections. 
 
-### Prerequisites
+### Features
 
-#### Host prerequisites
+This how-to guide uses the following {{site.prodname}} features:
 
-- Each Kubernetes host must have an IPv6 address that is reachable from
-  the other hosts.
-- Each host must have the sysctl setting `net.ipv6.conf.all.forwarding`
-  setting it to `1`.  This ensures both Kubernetes service traffic
-  and {{site.prodname}} traffic is forwarded appropriately.
-- Each host must have a default IPv6 route.
+- **IPAM**, a CNI plugin configuration with `assign_ipv6` and `assign_ipv4` flags
+- **IPPool**
 
-#### Kubernetes components prerequisites
+### Before you begin
 
-Kubernetes components must be configured to operate with IPv6.
-To enable IPv6, set the following flags.
+**{{site.prodname}} requirements** 
 
-##### kube-apiserver
+  {{site.prodname}} IPAM   
+  The defaut mode is **IPv4**.
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--bind-address` or `--insecure-bind-address` | Should be set to the appropriate IPv6 address or `::` for all IPv6 addresses on the host. |
-| `--advertise-address` | Should be set to the IPv6 address that nodes should use to access the kube-apiserver. |
-| `--service-cluster-ip-range` | Should be set to an IPv6 CIDR that will be used for the Service IPs, the DNS service address must be in this range. |
+**Kubernetes version requirements**
+  - For dual stack (alpha level), 1.16 and 1.17 
+  - 1.15 and earlier supports only one IP stack at a time (IPv6 or IPv4) 
 
-##### kube-controller-manager
+**Kubernetes IPv6 host requirements**
+  - An IPv6 address that is reachable from the other hosts
+  - The sysctl setting, `net.ipv6.conf.all.forwarding`, is set to `1`.    
+    This ensures both Kubernetes service traffic and {{site.prodname}} traffic is forwarded appropriately.
+  - A default IPv6 route
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--master` | Should be set with the IPv6 address where the kube-apiserver can be accessed. |
-| `--cluster-cidr` | Should be set to match the {{site.prodname}} IPv6 IPPool. |
+### How to
 
-##### kube-scheduler
+>**Note**: The following tasks work only for new clusters; you cannot migrate existing IPv4 clusters to IPv6.
+{: .alert .alert-info}
 
-| Flag | Value/Content |
-| ---- | ------------- |
-| `--master` | Should be set with the IPv6 address where the kube-apiserver can be accessed. |
+**Manifest install**
+- [Enable dual stack, Kubernetes](#enable-dual-stack-kubernetes)
+- [Enable IPv6-only, initial install, Kubernetes](#enable-ipv6-only-initial-install-kubernetes)
+- [Enable IPv6-only, after install, Kubernetes](#enable-ipv6-only-after-install-kubernetes)
```

Comment:
I don't understand why this is here, because your note just above says we can't migrate an existing cluster, and I believe that's correct.  Recommend deleting this title and the corresponding section below.

Reply:
Also "Kubernetes" is the wrong discriminator, because everything on this page is about Kubernetes.  Should be "manifest".

Reply:
Sorry for that...a leftover as doc team was trying to find how we designate "manifest" from "operator". We have simplified.